content/src/fieldConfig/profile.json

```diff
@@ -179,6 +185,7 @@
       "naicsCode": {
         "default": {
           "header": "NAICS Code",
+          "headerContextualInfo": "naics",
           "editText": "Edit",
           "addText": "Add",
```

Comment:
There is some styling issue with `Add`. It's not aligned with `NAICS Code`.

Reply:
So their are actually a lot of minor inconsistencies from design to actual at the moment, and I'm planning on a follow up with the design to team for a story to address all of the minor changes (figured it would be easier and not have this story balloon into fixing a bunch of styling)